JS-Algorithms/challenges.js

```diff
@@ -13,9 +13,20 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  if (typeof seconds !== "number" || seconds < 0) {
+    return "Invalid input: Please provide a non-negative number of seconds.";
+  }
+  const HH = String(Math.floor(seconds / 3600)).padStart(2, "0");
+  const MM = String(Math.floor((seconds % 3600) / 60)).padStart(2, "0");
+  const SS = String(seconds % 60).padStart(2, "0");
```

Comment:
Small nit: use caps for local variables

```suggestion
  const hh = String(Math.floor(seconds / 3600)).padStart(2, "0");
  const mm = String(Math.floor((seconds % 3600) / 60)).padStart(2, "0");
  const ss = String(seconds % 60).padStart(2, "0");
```